projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -185,13 +185,76 @@ public function build( $id ) {
 
 			// if task can't be found don't add anything
 			if ( ! empty( $task ) ) {
-				$tasks_for_task_list[] = $task;
+				$tasks_for_task_list[] = $this->build_task( $task );
 			}
 		}
 
 		return $tasks_for_task_list;
 	}
 
+	/**
+	 * Builds a single task with current state
+	 *
+	 * @param Task $task Task definition.
+	 * @return Task Task with current state.
+	 */
+	private function build_task( $task ) {
+		$built_task                 = array(
+			'id'    => $task['id'],
+			'title' => $task['title'],
+		);
+		$built_task['completed']    = $this->is_task_complete( $task );
+		$built_task['disabled']     = $this->is_task_disabled( $task );
+		$built_task['subtitle']     = $this->load_subtitle( $task );
+		$built_task['isLaunchTask'] = isset( $task['isLaunchTask'] ) ? $task['isLaunchTask'] : false;
+		$built_task['badge_text']   = isset( $task['badge_text'] ) ? $task['badge_text'] : '';
+		return $built_task;
+	}
+
+	/**
+	 * Loads a subtitle for a task, calling the callback if it exists.
+	 *
+	 * @param Task $task A task definition.
+	 * @return string The subtitle for the task.
+	 */
+	private function load_subtitle( $task ) {
+		if ( isset( $task['subtitle'] ) ) {
+			if ( is_callable( $task['subtitle'] ) ) {
+				return call_user_func( $task['subtitle'] );
```

Comment:
`NB:`

Does PHP not allow for invocation of the anonymous function like so?

`$task['is_disabled']( )`